devices/bosch.js

```diff
@@ -459,6 +579,49 @@ const fzLocal = {
 };
 
 const definition = [
+    {
+        zigbeeModel: ['RBSH-MMS-ZB-EU'],
+        model: 'BMCT-SLZ',
+        vendor: 'Bosch',
+        description: 'Bosch Light/shutter control unit II',
+        fromZigbee: [fz.on_off, fz.power_on_behavior, fz.electrical_measurement, fz.metering, fz.cover_position_tilt, fzLocal.bmct ],
+        toZigbee: [tzLocal.bmct_on_off, tzLocal.bmct, tz.cover_position_tilt, tz.cover_state, tz.power_on_behavior, tz.on_off ],
+        meta: {multiEndpoint: true, multiEndpointEnforce: true},
+        endpoint: (device) => {
+            return {'left': 2, 'right': 3};
+        },
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint1 = device.getEndpoint(1);
+            await reporting.bind(endpoint1, coordinatorEndpoint, [ 'genIdentify', 'closuresWindowCovering', 'haElectricalMeasurement', 64672]);
+            await reporting.currentPositionLiftPercentage(endpoint1);
+            await reporting.currentPositionTiltPercentage(endpoint1);
+            await endpoint1.read(64672, [0x0000, 0x0001, 0x0002, 0x0003, 0x0008, 0x0013, ], boschManufacturer);
+            const endpoint2 = device.getEndpoint(2);
+            await endpoint2.read(64672, [0x0008], boschManufacturer);
+            await reporting.bind(endpoint2, coordinatorEndpoint, ['genIdentify', 'genOnOff']);
+            await reporting.onOff(endpoint2);
+            const endpoint3 = device.getEndpoint(3);
+            await endpoint3.read(64672, [0x0008], boschManufacturer);
+            await reporting.bind(endpoint3, coordinatorEndpoint, ['genIdentify', 'genOnOff']);
+            await reporting.onOff(endpoint3);
+        },
+        exposes: [
+            exposes.enum('device_type', ea.STATE_SET, ['Light', 'Shutter']).withDescription('Device type: '),
```

Comment:
Instead of `['Light', 'Shutter']` you can use `Object.keys(stateDeviceType)` here (same for `switch_type`)